google/docs/tool.gpt

```diff
@@ -1,18 +1,38 @@
 ---
-Name: Google Docs 
+Name: Google Docs
 Metadata: bundle: true
 Description: Tools for managing Google Docs
-Share Tools: Read Doc
+Share Tools: Read Google Doc, Update Google Doc
```

Comment:
```suggestion
Share Tools: Read Google Doc, Update Google Doc, Create Google Doc
```

Reply:
I left this out on purpose because it's not really useful on its own.

Reply:
I added it to the bundle